redirect/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta http-equiv="Refresh" content="0; url='https://developer.shopware.com/resources/meteor-icon-kit/'"/>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Meteor icon kit</title>
+</head>
+<body>
+<h1>Redirecting....</h1>
+<p>
+    To open the Meteor icon kit, please follow <a href="https://developer.shopware.com/resources/meteor-icon-kit/">this link</a>!
```

Comment:
Just an idea: Should we add a automatic JS redirect after 5 seconds?

Reply:
See line #3, the redirect should be instant, that's just a fallback.